### PR TITLE
add Fibrous Finance

### PR DESCRIPTION
**Pull request by 0xKermo**

### CODE REVIEW

File: data/fibrousfinance.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "twitterName": "FibrousFinance",
+  "teamInfo": {
+    "founded": "2022-04-01T00:00:00.000Z",
+    "anonymous": false
+  },
+  "tokens": [],
+  "media": {
+    "logoUrl": "https://i.hizliresim.com/5ne373k.jpg",
+    "bannerUrl": "https://i.hizliresim.com/r5uy2xt.png",
```
Comment (ismaeldm95) [inline data/fibrousfinance.json:36]: It looks like this at the moment:

Screenshot 2022-10-06 at 16 53 39

Not sure if you guys are happy with the way the banner image looks like (pixelated, a bit zoomed in)

**Review by ismaeldm95 — CHANGES_REQUESTED** [review]
Thanks for adding fibrous! happy to see it included 💪
I added a few comments to make it work

File: data/fibrousfinance.json (hunk 1 above)
Comment (0xKermo) [reply data/fibrousfinance.json]: I fixed it and sent it

File: data/fibrousfinance.json [hunk 2]
```diff
@@ -32,9 +32,9 @@
   },
   "tokens": [],
   "media": {
-    "logoUrl": "https://i.hizliresim.com/5ne373k.jpg",
-    "bannerUrl": "https://i.hizliresim.com/r5uy2xt.png",
-    "previewUrl": "https://i.hizliresim.com/r5uy2xt.png",
+    "logoUrl": "https://github.com/Fibrous-Finance/fibrous-landing/blob/main/src/assets/ffpp.png",
+    "bannerUrl": "https://github.com/Fibrous-Finance/fibrous-landing/blob/main/src/assets/ffbanner.jpg",
+    "previewUrl": "https://github.com/Fibrous-Finance/fibrous-landing/blob/main/src/assets/ffbanner.jpg",
```
Comment (ismaeldm95) [inline data/fibrousfinance.json:37]: These new URLs don'e seem to work properly:
Screenshot 2022-10-10 at 12 26 58

**Review by ismaeldm95 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**